implementing initial vaccine data fixtures.

tiemfah opened the pull request:
So that we don't have to type in vaccine model every time.
type this after we run migration.
```bash
python manage.py loaddata vaccine/fixtures/vaccine.json
```

tiemfah committed:
implementing initial vaccine data fixtures.

tiemfah commented:
need a test case to test that loaddata works.

chiiida reviewed (changes requested):
What about DoseModel? Can you also add JSON for DoseModel?
We also need DoseModel in order to test the whole model of vaccine and user.

chiiida committed:
Add field in dose model

chiiida committed:
Make migration for vaccine and dose models

tiemfah committed:
Update README.md

Add instruction on how to config Google credential and how to use premade data.

tiemfah commented:
Will merge into master directly after @llleyelll  merge her `vaccine-model-upgrade` branch.
